components/Panel/Panel.tsx

```diff
@@ -29,20 +28,18 @@ export function Panel() {
   const panelWidth = usePanelWidth();
   const contentRef = useRef<HTMLDivElement | null>(null);
 
+  const transitions = useTransition(panelOpen, {
```

Comment:
we're still using react spring's transition helper for animating mount and unmount of the panel's overlay as before.